SEED-platform/seed
# feat: added delete org functionality for non-admins #2542

#### What's this PR do?
- Adds the ability for users to delete organizations and sub-organizations that they own
- Reorganized org settings into vertical tabs

#### How should this be manually tested?
- Attempt to delete an org that you own
  - Once it starts the modal cannot be dismissed
  - On success the page will fully reload to the home page (since the page you were on will no longer be accessible)
- Compare the text when deleting orgs vs sub-orgs, it should be specific to the current type
- Ensure that non-owners of orgs and non-owners of sub-orgs cannot delete orgs they belong to (either via UI or API)

#### What are the relevant tickets?
#2542 

#### Screenshots (if appropriate)
New, better-organized settings page:
![image](https://user-images.githubusercontent.com/411466/111557838-10462580-8753-11eb-9b65-63beae080f9f.png)

Delete modal:
![image](https://user-images.githubusercontent.com/411466/111557863-248a2280-8753-11eb-9ebf-88ce1745a12d.png)
